DFReader.py

```diff
@@ -166,7 +166,7 @@ def set_mult_ids(self, mult_ids, mult_lookup):
         for i in range(len(self.units)):
             # If the format has its own multiplier, do not adjust the unit,
             # and if no unit is specified there is nothing to adjust
-            if self.msg_mults[i] is not None or self.units[i] == "":
+            if self.msg_mults[i] is not None or self.units[i] != "":
```

Comment:
I think you need just:
```suggestion
            if self.msg_mults[i] is not None:
```
Otherwise, things like *.TimeUS will say its in 's' instead of 'μs' and PM.Load will say its in '%' instead of 'd%', when running 'dump PM --verbose'.